src/components/TrackTile/TrackerDetails/AdvancedTrackerDetails/AdvancedTrackerDetails.tsx

```diff
@@ -264,16 +265,17 @@ export const AdvancedTrackerDetails = (props: AdvancedTrackerDetailsProps) => {
   );
 
   return (
-    <ScrollView>
-      <DayPicker
-        dateRange={dateRange}
-        tracker={tracker}
-        color={tracker.color}
-        target={targetAmount}
-        unit={selectedUnit}
-        onChange={setDateRange}
-      />
-
+    <ScrollView contentContainerStyle={styles.container}>
+      <View style={{ alignItems: 'center' }}>
```

Comment:
This might need a named style. I think we disallowed inline styles

Reply:
Yes, thanks, I meant to move that down to named styles.